stringfuzz/generators/regex.py

```diff
@@ -70,24 +100,32 @@ def make_regex_string(min_length, max_length):
 
     return smt_str_to_re(smt_str_lit(string))
 
-def make_random_term(depth):
+def make_random_term(depth, operator=None):
```

Comment:
Although this works, I feel like it's a bit complex to understand exactly how the operators alternate.  What do you think of:
- giving this function an index into `_operator_list` (so, `operator_index` instead of `operator` here),
- changing `next_operator` to `get_operator`, which just does a modulo access, and then
- just incrementing the index in recursive calls (like the depth)?